corehq/apps/cachehq/cachemodels.py

```diff
@@ -48,10 +48,10 @@ class ReportGenerationCache(GenerationCache):
     generation_key = '#gen#reports#'
     doc_types = ['ReportConfig', 'ReportNotification']
     views = [
+        "reportconfig/all_notifications",
         'reportconfig/configs_by_domain',
         'reportconfig/notifications_by_config',
         "reportconfig/user_notifications",
-        "reportconfig/daily_notifications",
```

Comment:
I went through commit by commit and saw this one

> rename daily_notifications to all_notifications (sort alphabetically)

this is actually just removing a non-existent view (`reportconfig/daily_notifications`) and adding in an existing but previously uncovered view (`reportconfig/all_notifications`)? The 'rename' framing threw me off.

Reply:
Ah good point. I am not sure why I immediately assumed the view was renamed, but yes you interpreted that correctly.